charts/victoria-metrics-agent/values.yaml

```
@@ -129,6 +129,11 @@ extraContainers: []
 #   - name: config-reloader
 #     image: reloader-image
 
+initContainers: 
+  {}
```

```suggestion
  []
```

As far as I can see, the default value here should be an empty list?